google/cloud/pubsub/internal/tracing_batch_callback.cc

```diff
@@ -93,6 +93,7 @@ class TracingBatchCallback : public BatchCallback {
       if (kv.second.subscribe_span) kv.second.subscribe_span->End();
       if (kv.second.concurrency_control_span)
         kv.second.concurrency_control_span->End();
```

Comment:
Nit from the past:  We typically use `{}`s when the entire `if` doesn't fit on one line.

Reply:
Done.